src/components/bar/Bar.tsx

```diff
@@ -0,0 +1,7 @@
+import { BarWrapper } from './Bar.style';
+
+const Bar = () => {
+  return <section css={BarWrapper}></section>;
```

Comment:
p4: 여기는 왜 section 태그를 사용하신 걸까요? 
그리고 태그 내부에 컨텐츠 없이 독립적으로 사용되면 아래와 같이 사용할 수 있어요!
```suggestion
  return <section css={BarWrapper} />;
```

Reply:
맞아요! 저도 같은 생각을 했는데 전체적인 코드 스타일을 맞출까해서 
```tsx
return <section></section>
```
처럼 했던 것 같아요!
생각해보니 독립적 사용일 때에는 도영님 처럼 사용한게 좋을 것 같네요!

또한 단순히 밑줄 느낌의 태그는 뭘 사용해야할까 고민하던중 section을 사용하게 된건데, 도영님이 생각하시기에는 어떤 태그가 어울릴지 의견 주시면 감사드리겠습니다!!

Reply:
우선 div로 수정했습니다!

Reply:
음 뭔가 섹션은 안에 내용이 들어가는 느낌이라서...! 저도 div로 사용할 것 같네요!!